lib/std/Build/Step/Run.zig

```diff
@@ -592,6 +608,12 @@ fn make(step: *Step, prog_node: std.Progress.Node) !void {
     var man = b.graph.cache.obtain();
     defer man.deinit();
 
+    for (run.lazy_env_map.items) |lazy_env_var| {
+        const path = lazy_env_var.value.getPath2(b, step);
+        run.setEnvironmentVariable(lazy_env_var.key, path);
```

Comment:
this function must be called only during the configure phase. consider that the make function can be re-executed multiple times, so it should not call this every time.